src/components/TileGrid/Tile/RectTile.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from "styled-components";
+import { ReactComponent as EduLogo } from "../../../assets/svgs/edu.svg";
+import one from "../../../assets/images/flyer.png";
+import two from "../../../assets/images/arduino.png";
+import three from "../../../assets/images/cscareers.png";
+import four from "../../../assets/images/donuts.png";
+import five from "../../../assets/images/interview.png";
+import six from "../../../assets/images/sweacmfinal.png";
+
+interface propTypes {
+  title: string;
+  date: string;
+  time: string;
+  num: number;
+}
+
+const RectTile = (props: propTypes) => {
+  return (
+    <TileComponent tabIndex={props.num}>
```

Comment:
The `tabIndex` was a way to inject the random number into the styled div. Will also be gone once we have real data.